services/wallet/api.go

```diff
@@ -248,6 +260,7 @@ func (api *API) GetPendingTransactionsForIdentities(ctx context.Context, identit
 	return
 }
 
+// @deprecated
 // TODO - #11861: Remove this and replace with EventPendingTransactionStatusChanged event and Delete to confirm the transaction where it is needed
 func (api *API) WatchTransactionByChainID(ctx context.Context, chainID uint64, transactionHash common.Hash) (err error) {
```

Comment:
This is still used by Nim `transaction.Service` for watching a transaction. The [#11861](https://github.com/status-im/status-desktop/issues/11861) is supposed to address this legacy API and replace it with the new event-based.

Reply:
So that means it is deprecated? still in use but nevertheless shouldn't be right?

Reply:
Yes, this should go away.